CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 This changelog keeps track of all changes to the Packs SDK. We follow conventions from [keepachangelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+### Added
+
+- Added "useProofKeyForCodeExchange" option to OAuth2 authentication to support PKCE extention. While it's optionally supported by most OAuth2 providers, it might be required by some websites (e.g. Twitter).
```

Comment:
hmm wait im not sure how we got a 0.11.0 header even tho it's not released yet but do you want to move it into 0.11.0?

Reply:
oh this is from jonathan's last change. Wonder if we should include this in there too but not big deal

Reply:
I don't want to further block 0.11.0 release. This might go into 0.12.0.